HoloPipelines/models/dense_vnet_abdominal_ct/Dockerfile

```diff
@@ -0,0 +1,44 @@
+#Download base image ubuntu 18.04
+FROM nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
+
+
+# Set variables
+ARG APP_DIR=/usr/src/app
+ARG PORT=5000
+
+# Update Software repository, install package and clean the package 
+RUN apt-get update  && \
+    apt-get install -y --no-install-recommends python3-pip python3-dev build-essential nano &&\
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+
+
+# Copy the flask script into the container and set the working directory
+RUN mkdir -p ${APP_DIR}
+WORKDIR ${APP_DIR}
+COPY server.py test.py ./
+
+# Copy the requirements file in to image for pip download
+COPY requirements.txt ./
+
+# Run install tensorflow, niftynet, flask
+RUN pip3 install --upgrade pip setuptools
+RUN pip3 install wheel
+RUN pip3 install -r requirements.txt
+
+# Install abdominal model
+RUN net_download dense_vnet_abdominal_ct_model_zoo
```

Comment:
When I try to build the image on my computer (WSL Ubuntu in Windows 10), it crashes at this point with the stacktrace below. Is this the error you encountered as well @ansonwong9695 ?

<details>
<summary>Show stacktrace</summary>

```shell
Step 12/17 : RUN net_download dense_vnet_abdominal_ct_model_zoo
 ---> Running in 9a532f38e74b
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/usr/lib/python3.5/imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "/usr/lib/python3.5/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: libcuda.so.1: cannot open shared object file: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/niftynet/__init__.py", line 15, in <module>
    import tensorflow as tf
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/__init__.py", line 24, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-import
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/pywrap_tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/usr/lib/python3.5/imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "/usr/lib/python3.5/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: libcuda.so.1: cannot open shared object file: No such file or directory


Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/errors

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/net_download", line 6, in <module>
    from niftynet.utilities.download import main
  File "/usr/local/lib/python3.5/dist-packages/niftynet/__init__.py", line 17, in <module>
    raise ImportError('NiftyNet is based on TensorFlow, which'
ImportError: NiftyNet is based on TensorFlow, which does not seem to be installed on your system.

Please install TensorFlow (https://www.tensorflow.org/) to be able to use NiftyNet.
The command '/bin/sh -c net_download dense_vnet_abdominal_ct_model_zoo' returned a non-zero code: 1
```
</details>

Reply:
I think that is because you have not installed the Nvidia docker https://github.com/NVIDIA/nvidia-docker, 
after you installed you might also need to modify a JSON file by following this directory
 ```/etc/docker/daemon.json```
the json should be modified in this way
```
{
"default-runtime": "nvidia",
  "runtimes": {
     "nvidia": {
         "path": "/usr/bin/nvidia-container-runtime",
         "runtimeArgs": []
     }
 }

}
```
after the modification, you might need to restart the docker by running this two command lines
```
1. systemctl stop docker
2. systemctl start docker
```
you can test your modification by running this command line
```
docker run  --rm nvidia/cuda nvidia-smi
````
but I am not sure, this can solve your problem or not

Reply:
Thanks Kawai, I will try tomorrow! If it works I will also copy your instructions to the `README`, this is the stuff we should document.